docs/data/material/components/autocomplete/GoogleMaps.tsx

```diff
@@ -144,9 +144,9 @@ export default function GoogleMaps() {
           option.structured_formatting.main_text,
           matches.map((match: any) => [match.offset, match.offset + match.length]),
         );
-
+        const {key, ...nonKeyProps} = props as any;
```

Comment:
Let's remove `as any` and wait for https://github.com/mui/material-ui/pull/42689, which fixes the `props` type.

```suggestion
        const {key, ...nonKeyProps} = props;
```

Reply:
Done facd8fbb2d